wrapanapi/systems/rhevm.py

```diff
@@ -649,6 +649,17 @@ def deploy(self, vm_name, cluster, timeout=900, power_on=True, **kwargs):
             vm.start()
         return vm
 
+    def list_disks(self, disk_id=None):
+        """Return a list of disks in the template
+
+        Args:
+            disk_id (bool) -- if True, returns ids of the disks, else returns disk names
+        """
+        return [
+            disk.id if disk_id is True else disk.name
```

Comment:
`disk.id if disk_id else disk.name` would suffice, but being extra-explicit doesn't hurt. :)

Reply:
`getattr(disk, 'id' if disk_id else 'name')` too ..... dunno which I like the most.